src/libraries/System.Drawing.Primitives/src/System/Drawing/Color.cs

```diff
@@ -487,12 +487,42 @@ private void GetRgbValues(out int r, out int g, out int b)
             b = (int)(value & ARGBBlueMask) >> ARGBBlueShift;
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private void MinMaxRgb(out int min, out int max, int r, int g, int b)
+        {
+            if (b < g) {
```

Comment:
Move trailing braces to their own lines for consistency with existing .NET code.

Reply:
Thanks, updated.